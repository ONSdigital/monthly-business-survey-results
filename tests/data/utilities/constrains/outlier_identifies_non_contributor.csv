reference,period,spp_form_id,questioncode,value,outlier_weight
1,202001,14,42,900,0.7
1,202001,14,43,0,0.8
1,202001,14,40,900,0.9
