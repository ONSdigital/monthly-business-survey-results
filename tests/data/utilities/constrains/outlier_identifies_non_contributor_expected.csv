reference,period,spp_form_id,questioncode,value,outlier_weight,winsorised_value,post_winsorised
1,202001,14,42,900,0.7,630.0,FALSE
1,202001,14,43,0,0.8,0.0,FALSE
1,202001,14,40,900,0.7,630.0,TRUE
